src/js/history.js

```diff
@@ -50,3 +54,15 @@ export const addHistory = async (newHistoryFactors) => {
   }
   chrome.storage.local.set({ tabKillerHistory: history });
 };
+
+const allClear = async () => {
+  const language = (await getLocalStorage('tabKillerLanguage')) || 'ja';
+  const confirmMessage =
+    language === 'ja' ? '本当にすべて削除しますか？' : 'Can I delete All?';
```

Comment:
削除の主体はユーザなのでyouじゃないかと
```suggestion
    language === 'ja' ? '本当にすべて削除しますか？' : 'Are you sure you want to delete all history?';
```

Reply:
copilotくんによく言いつけとく